classes/exception/PrestaShopException.php

```diff
@@ -34,6 +34,10 @@ class PrestaShopExceptionCore extends Exception
      */
     public function displayMessage()
     {
+        if (getenv('kernel.environment') === 'test') {
+            throw $this;
```

Comment:
Being re-thrown, the exception stack trace is be formatted by phpunit thanks to its internals formatter.